packages/storybook/cxl-ui/cxl-card/[theme=cxl-course].stories.js

```diff
@@ -0,0 +1,67 @@
+import { html } from 'lit';
+import '@conversionxl/cxl-ui/src/components/cxl-card.js';
+import '@conversionxl/cxl-lumo-styles';
+
+export default {
+  title: 'CXL UI/cxl-card',
+  parameters: {
+    layout: 'centered',
+  },
+};
+
+const Template = ({ el, State }) => html`
+  <cxl-card
+    id="${el.cxl_hybrid_attr_post['@attributes'].id}"
+    class="${el.cxl_hybrid_attr_post['@attributes'].class}"
+    theme="cxl-course cxl-course-${State}"
+  >
+    <header class="entry-header">
+      ${State === 'new'
+      ? html`<vaadin-icon icon="cxl:new"></vaadin-icon>`
+      : html``}
+      <div>
+        <div class="entry-title">
+          Account based marketing
+          ${State === 'completed'
+            ? html`<vaadin-icon icon="lumo:checkmark"></vaadin-icon>`
+            : html``}
+        </div>
+        <div class="entry-byline">
+          <div><vaadin-icon icon="lumo:clock"></vaadin-icon>3h 00min</div>
+          <div>By: Ton Wesseling</div>
+        </div>
+        ${State === 'with-roadmaps'
+          ? html`
+            <div class="entry-summary">
+              <span theme="badge primary pill">Team Roadmap</span>
+              <span theme="badge secondary pill">Personal Roadmap</span>
+            </div>`
+          : html``}
+      </div>
+      <img
+        class="thumbnail"
+        src="https://cxl.com/institute/wp-content/uploads/2020/05/48192546_10156982340630746_8127333122065825792_n-wpv_400pxx400px_center_center.jpg"
+      />
+    </header>
+  </cxl-card>
+`;
+
+export const CXLCourse = Template.bind({});
+
+CXLCourse.args = {
+  el: { cxl_hybrid_attr_post: { '@attributes': {} } },
+  State: 'default',
+};
+
+CXLCourse.argTypes = {
+  State: {
+    control: {
+      options: ['default', 'new', 'completed', 'with-roadmaps'],
```

Comment:
I suggest to change `with-roadmaps` to `with-badges` to make it more universal as badges might be used also for tags etc.

And question: is it possible to apply multiple states at the same time?

Reply:
Changed the roadmaps -> badges and now it's possible to select multiple [states](https://deploy-preview-289--conversionxl-aybolit.netlify.app/?path=/docs/cxl-ui-cxl-card--cxl-course)